### PR TITLE
New Administration chapter

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
This PR collects the various administrative sections currently in Cypher Manual and places them under a new chapter heading.
This includes:
- database management
- database alias management
- server management
- access control

Also included in PR:
- xref redirects

Corresponding PRs:
Operations Manual: https://github.com/neo4j/docs-operations/pull/635
Cheat sheet: https://github.com/neo4j/docs-cheat-sheet/pull/71

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-494.surge.sh

**Review by davidoliverSP2 — APPROVED** [review]
This looks thorough, and a really nice improvement - thanks @JPryce-Aklundh 

Be mindful to not add too much to this big PR, any further editorial changes can go in afterwards.

File: modules/ROOT/pages/administration/aliases.adoc [hunk 1]
```diff
@@ -905,7 +905,7 @@ CREATE ALIAS garden.trees FOR DATABASE trees AT 'neo4j+s://location:7687' USER a
 ////
 
 Database aliases can be altered using `ALTER ALIAS` to change its database target, properties, url, user credentials, or driver settings.
-The required privileges are described xref::access-control/dbms-administration.adoc#access-control-dbms-administration-alias-management[here].
+The required privileges are described xref:administration/access-control/dbms-administration.adoc#access-control-dbms-administration-alias-management[here].
```
Comment (Hunterness) [inline modules/ROOT/pages/administration/aliases.adoc]: why removing the `:`? all other places got to keep theirs?

File: modules/ROOT/pages/clauses/index.adoc [hunk 2]
```diff
@@ -12,13 +12,13 @@ This section contains information on all the clauses in the Cypher query languag
 [[administration-clauses]]
 == Administration clauses
 
-These comprise clauses used to manage databases, schema and security; further details can found in xref::databases.adoc[Database management] and xref::access-control/index.adoc[Access control].
+These comprise clauses used to manage databases, schema and security; further details can found in xref::administration/databases.adoc[Database management] and xref::administration/access-control/index.adoc[Access control].
```
Comment (Hunterness) [inline modules/ROOT/pages/clauses/index.adoc:15]: Maybe not part of the restructure as such but should this section also get the alias and server commands?

File: modules/ROOT/pages/administration/access-control/dbms-administration.adoc [hunk 3]
```diff
@@ -71,13 +71,13 @@ This section explains how to use Cypher to manage Neo4j DBMS administrative priv
 
 All DBMS privileges are relevant system-wide.
 Like user management, they do not belong to one specific database or graph.
-For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/neo4j-databases-graphs.adoc[].
+For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/cypher_neo4j.adoc[].
```
Comment (Hunterness) [inline modules/ROOT/pages/administration/access-control/dbms-administration.adoc:74]: why switching from `neo4j-databases-graphs`? that is still a document (you have edited it) and it contains the differences

File: modules/ROOT/pages/administration/access-control/dbms-administration.adoc [hunk 4]
```diff
@@ -71,13 +71,13 @@ This section explains how to use Cypher to manage Neo4j DBMS administrative priv
...
 
 image::privileges_grant_and_deny_syntax_dbms_privileges.svg[title="Syntax of GRANT and DENY DBMS Privileges"]
 
 image::privileges_hierarchy_dbms.svg[title="DBMS privileges hierarchy"]
```
Comment (Hunterness) [inline modules/ROOT/pages/administration/access-control/dbms-administration.adoc:78]: These images don't seem to render :(

**Review by Hunterness — COMMENTED** [review]
Looks good in general, just some questions/comments

(I didn't try clicking all the links, I assume you have tested them out XD)

I still find it curious that we are now re-introducing a layer we previously removed to not get to deep stacks XD

File: modules/ROOT/pages/administration/access-control/dbms-administration.adoc (hunk 3 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/administration/access-control/dbms-administration.adoc:74]: Some of the old pages for the introduction are currently still in the repo - they will be deleted as soon as I am done with the necessary restructuring stages.

File: modules/ROOT/pages/clauses/index.adoc (hunk 2 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/clauses/index.adoc:15]: Sounds reasonable (but not in this PR) :bowtie:

File: modules/ROOT/pages/administration/access-control/dbms-administration.adoc (hunk 4 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/administration/access-control/dbms-administration.adoc:78]: Good spot - they don't render in the published docs either :/ will look into it!

**Comment by JPryce-Aklundh** [comment]
@Hunterness I understand - I can't really answer to prior restructuring decisions. To me, it makes sense to have these 4 sections within a broader Administration chapter, and I don't think it produces overly deep stacks. This change will also alleviate the likely move of all these sections to the Operations Manual (I cannot say when that will happen, but we are starting planning the migration in earnest later today).

**Review by Hunterness — APPROVED** [review]

**Pull request merged**